c/models/column_hasher.cc

```diff
@@ -20,46 +20,25 @@
 // IN THE SOFTWARE.
 //------------------------------------------------------------------------------
 #include "models/column_hasher.h"
+#include "utils/assert.h"
```

Comment:
Any reason to include `assert.h`?

Reply:
I added an `xassert()` at some point to test the changes, but then forgot to remove the header after the assert was no longer necessary.
So, not really needed, can be removed anytime (but at least it doesn't hurt).

Reply:
Yeah, that's what I thought.